crates/nu-protocol/src/signature.rs

```diff
@@ -154,6 +154,12 @@ impl Signature {
         self
     }
 
+    /// Add extra usage info to the signature
+    pub fn extra_usage(mut self, extra_usage: impl Into<String>) -> Signature {
```

Comment:
I'm not sure we need to change the signature either. Do we?

Reply:
Do you know why there is an extra_usage field in the Signature struct?

Reply:
I was just looking at how other commands implement `extra_usage` like `ansi` and it works fine before this change. So, that's why I was questioning it.

Reply:
@royb3 The field in Signature is there so that we can pass this info without having the Command around. As far as I understand, it is useful in plugins.